src/EdFi.AnalyticsMiddleTier.DataStandard2/Base/MSSQL/0046-View-SectionDim-Alter.sql

```diff
@@ -0,0 +1,94 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+IF EXISTS (
+        SELECT 1
+        FROM INFORMATION_SCHEMA.VIEWS
+        WHERE TABLE_SCHEMA = 'analytics'
+            AND TABLE_NAME = 'SectionDim'
+        )
+BEGIN
+    DROP VIEW analytics.SectionDim
+END
+GO
+
+CREATE VIEW [analytics].[SectionDim]
+AS
+SELECT CAST(s.SchoolId AS VARCHAR) AS SchoolKey
+    ,CONCAT (
+        CAST(s.SchoolId AS NVARCHAR)
+        ,'-'
+        ,[s].[ClassPeriodName]
+        ,'-'
+        ,[s].[ClassroomIdentificationCode]
+        ,'-'
+        ,s.[LocalCourseCode]
+        ,'-'
+        ,CAST([s].[TermDescriptorId] AS NVARCHAR)
+        ,'-'
+        ,CAST(s.[SchoolYear] AS NVARCHAR)
+        ,'-'
+        ,s.[UniqueSectionCode]
+        ,'-'
+        ,CAST(s.[SequenceOfCourse] AS NVARCHAR)
+        ) AS [SectionKey]
+    ,CONCAT (
+        [Descriptor].[Description]
+        ,'-('
+        ,s.[LocalCourseCode]
+        ,')'
+        ,'-'
+        ,[Course].[CourseTitle]
+        ,'-('
+        ,s.[ClassPeriodName]
+        ,')-'
+        ,TermType.Description
+        ) AS Description
+    ,CONCAT (
+        [s].[LocalCourseCode]
+        ,'-'
+        ,COALESCE([Course].[CourseTitle], '')
+        ) AS [SectionName]
+    ,Session.SessionName
+    ,[CourseOffering].[LocalCourseCode]
+    ,CAST(s.SchoolYear AS VARCHAR) as SchoolYear
+    ,EducationalEnvironmentType.Description AS EducationalEnvironmentDescriptor
+    ,COALESCE(CAST(sch.LocalEducationAgencyId AS VARCHAR),'') AS LocalEducationAgencyKey
```

Comment:
I don't think this `COALESCE` here is necessary. Actually, I think the left join we have on line 80 can be inner join since `[Course].[EducationOrganizationId]` can't be null.